frontend/integration-tests/tests/overview/overview.scenario.ts

```diff
@@ -38,7 +38,7 @@ describe('Visiting Overview page', () => {
 
       it(`displays a ${kindModel.id} in the project overview list`, async () => {
         await browser.wait(until.presenceOf(overviewView.projectOverview));
-        await overviewView.itemsAreVisible();
+        await overviewView.testObjectVisible(resourceName);
```

Comment:
Is there a reason for this change?

I'm a little worried that this will make debugging test failures harder because when `browser.wait()` fails we lose the stack trace. So it won't be clear what line failed if the present isn't present. It also seems to be duplicating the check just below.

Reply:
Running `bridge` on host, instead of inside VM/Docker, helped a lot in bringing consistency for my local tests. Turned out I didn't need this change, as original code worked reliably with FF.